charts/vigil/templates/configmap.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "vigil.fullname" . }}
+  labels:
+    {{- include "vigil.labels" . | nindent 4 }}
+data:
+  vigil.cfg: |
+    [server]
+
+    log_level = "{{ .Values.vigil.log_level }}"
+    inet = "0.0.0.0:8080"
+    workers = 4
+    reporter_token = "{{ .Values.vigil.token }}"
+
+    [assets]
+
+    path = "./res/assets/"
+
+    [branding]
+
+    page_title = "{{ .Values.vigil.page_title }}"
+    page_url = "{{ .Values.vigil.page_url }}"
+    company_name = "{{ .Values.vigil.company_name }}"
+    icon_color = "{{ .Values.vigil.icon_color }}"
+    icon_url = "{{ .Values.vigil.icon_url }}"
+    logo_color = "{{ .Values.vigil.logo_color }}"
+    logo_url = "{{ .Values.vigil.logo_url }}"
+    website_url = "{{ .Values.vigil.website_url }}"
+    support_url = "{{ .Values.vigil.support_url }}"
+    custom_html = "{{ .Values.vigil.customer_html }}"
```

Comment:
probably set these to the sample [config.cfg](https://github.com/valeriansaliou/vigil/blob/master/config.cfg) that we have in this repo so that at least the app boots properly & healthiness/readiness probes pass.

At least the user will be able to port-forward to the service and see if the status page is working & how it looks initially (as a demo).